src/luxor_overrides.jl

```diff
@@ -240,6 +244,14 @@ draws the text from left to right in the first 15 frames and in the last 15 fram
 - `angle::Float64` defaults to `0.0` and specifies the angle of the text
 """
 function text(str, pos = O; valign = :baseline, halign = :left, angle = 0.0)
+    call_luxor_if_outside_rendering(
+        Luxor.text,
+        str,
+        pos;
+        valign = valign,
+        halign = halign,
+        angle = angle,
+    ) && return
```

Comment:
will this return what Luxor.text returns ? 
might not be a big issue , but if user  is relying on Luxor functions return value it would be nice to return what Luxor returns

Reply:
No it doesn't good point will change.

Reply:
Fixed